Shared/css/style.css

```diff
@@ -7069,13 +7069,27 @@ only screen and (max-device-width: 568px) {
 
 .fab-btn-list2 {
   position: absolute;
-  padding: 18px 0px 0px;
-  margin: 0px 0px 0px -3px;
-  left: auto;
+  padding: 28px 25px 0px 0px;
+  margin: 0px;
+  left: -55px;
```

Comment:
Up for discussion, but would like the use of REM instead of pixels for unit positionings > 4px.. Makes it more responsive